index.html

```diff
@@ -184,13 +184,49 @@ <h3>Exposing HTML Features That Do Not Directly Map to Accessibility APIs</h3>
         <abbr title="application programming interface">API</abbr> properties</a> in the [[core-aam-1.2]].
       </p>
     </section>
+    <section>
+      <h3 id="minimum-role">Exposing HTML Features That Require a Minimum Role</h3>
+      <p>
+        A <dfn>minimum role</dfn> is the equivalent WAI-ARIA role an element will map to if the element does not have a more specific implicit role or 
+        platform role mappings, e.g., a non-generic role. This can help ensure that users of assistive technologies get the best possible experience for 
+        commonly-used and valid HTML markup where otherwise a role would not be exposed.
+      </p>     
+      <p>
+        A minimum role is provided when all of the following conditions are true:
+      </p>
+      <ul>
+        <li>
+          the author has <strong>not</strong> specified a valid explicit WAI-ARIA role to the element, 
+          <strong>or</strong> the specified role is either `none`, `presentation`, or `generic`;
+        </li>
+        <li>
+          the element either has no implicit WAI-ARIA role or platform role mappings, or it has an implicit `generic` or `none` computed role; 
+        </li>
+        <li>
+          and the author has specified attributes which require a minimum role mapping for the element.
+        </li>
+      </ul>
+      <p>
+        The <a href=#html-attribute-state-and-property-mappings">HTML Attribute State and Property Mappings</a> section identifies the specific global attributes which would require
```

Comment:
and missing quote...
```suggestion
        The <a href="#html-attribute-state-and-property-mappings">HTML Attribute State and Property Mappings</a> section identifies the specific global attributes which would require
```